src/api/github/getRelevantCommit.ts

```diff
@@ -12,10 +12,9 @@ import { getClient } from '@api/github/getClient';
  * caused a bump commit in getsentry (whose commit message references the sentry commit).
  * In this case, it will return the sentry commit.
  */
-export async function getRelevantCommit(ref: string, client?: Octokit) {
```

Comment:
As with `getChangedStack`, we don't realize any performance gain from passing an octokit around vs. calling getClient to get the same object from the (new) cache. Here though we were actually using the parameter in two of the four calls, so that is undone.